AdyenDropIn/DropInComponent.swift

```diff
@@ -239,8 +241,6 @@ extension DropInComponent: ActionComponentDelegate {
     /// :nodoc:
     public func didFail(with error: Error, from component: ActionComponent) {
         if case ComponentError.cancelled = error {
-            paymentInProgress = false
-            stopLoading(withSuccess: false, completion: nil)
```

Comment:
StopLoading has to be called here for issuer list component and payment method list component to stop spinning if user cancels